src/node.js

```diff
@@ -73,6 +76,30 @@
         NativeModule.require('_third_party_main');
       });
 
+    } else if (process.openManPage) {
+      // Attempt to spawn `man 1 node`.
+      const cp = NativeModule.require('child_process');
```

Comment:
I plan to make all the large option cases into separate functions, probably in a different PR?